content/build/hydrobuild.md

```diff
@@ -451,3 +451,24 @@ default `docker` builder, run the following command:
 ```console
 $ docker context use default
 ```
+
+### How do I manage the build cache with Hydrobuild?
+
+You don't need to manage the builder's cache manually. The system manages it
+for you through [garbage collection](./cache/garbage-collection.md).
+
+Hydrobuild uses the following garbage collection limits:
+
+- Size: 90% of 1TB
+- Age: cache not used in the past 180 days
+- Number of builds: 10 000
+
+Old cache is automatically removed if you hit any of these limits.
+
+You can clear the builder's cache manually with the `docker buildx prune`
```

Comment:
Maybe add: You can check the state of the cache using `docker buildx du` command ?

Reply:
good idea